Source/WebCore/page/ContextMenuController.cpp

```diff
@@ -1795,8 +1799,7 @@ void ContextMenuController::showImageControlsMenu(Event& event)
 
 void ContextMenuController::performPDFJSAction(LocalFrame& frame, const String& action)
 {
-    auto* pdfDocument = dynamicDowncast<PDFDocument>(frame.ownerElement()->document());
-    if (pdfDocument)
+    if (RefPtr pdfDocument = dynamicDowncast<PDFDocument>(frame.ownerElement()->document()))
```

Comment:
I would have changed this to just `document` as a drive-by. My opinion is that the variable name doesn’t have to say what the type is, unless there’s some important distinction, like multiple variables pointing to the same object. It doesn’t make the code easier to read; words are easier to read than acronyms smashed together with words.

Reply:
Agreed. Fixing.